<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="SumathiLaptop" failures="7" tests="16" name="Default test" time="17.322" errors="0" timestamp="2023-08-18T12:26:01 MDT">
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="4.063"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.853">
    <failure type="java.lang.AssertionError" message="expected [400] but found [201]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [201]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at stepDefinitions.TC01_PostReq_Program.user_receives_failed_status_code(TC01_PostReq_Program.java:122)
at ✽.User receives failed Status Code(file:///C:/Users/nsuma/OneDrive/Desktop/Sample%20API/LMS_API_2023/src/test/java/features/TC01_PostProgram.feature:21)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.528"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.414"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.707"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.868"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.239">
    <failure type="java.lang.AssertionError" message="2 expectations failed.
Expected status code &amp;lt;400&amp;gt; but was &amp;lt;200&amp;gt;.

Expected header &amp;quot;Content-Type&amp;quot; was not &amp;quot;application/json&amp;quot;, was &amp;quot;text/plain;charset=UTF-8&amp;quot;. Headers are:
Server=Cowboy
Connection=keep-alive
Content-Type=text/plain;charset=UTF-8
Content-Length=50
Date=Fri, 18 Aug 2023 18:25:49 GMT
Via=1.1 vegur
">
      <![CDATA[java.lang.AssertionError: 2 expectations failed.
Expected status code <400> but was <200>.

Expected header "Content-Type" was not "application/json", was "text/plain;charset=UTF-8". Headers are:
Server=Cowboy
Connection=keep-alive
Content-Type=text/plain;charset=UTF-8
Content-Length=50
Date=Fri, 18 Aug 2023 18:25:49 GMT
Via=1.1 vegur

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.CallSiteArray.defaultCallConstructor(CallSiteArray.java:57)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:263)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:101)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:288)
at stepDefinitions.TC03_PutReq_ById_Prog.user_receive_http_status_code_and_error_message(TC03_PutReq_ById_Prog.java:118)
at ✽.User receive HTTP Status code and error message(file:///C:/Users/nsuma/OneDrive/Desktop/Sample%20API/LMS_API_2023/src/test/java/features/TC03_PutProgram.feature:16)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.387"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.033">
    <failure type="java.lang.AssertionError" message="2 expectations failed.
Expected status code &amp;lt;400&amp;gt; but was &amp;lt;200&amp;gt;.

Expected header &amp;quot;Content-Type&amp;quot; was not &amp;quot;application/json&amp;quot;, was &amp;quot;text/plain;charset=UTF-8&amp;quot;. Headers are:
Server=Cowboy
Connection=keep-alive
Content-Type=text/plain;charset=UTF-8
Content-Length=50
Date=Fri, 18 Aug 2023 18:25:51 GMT
Via=1.1 vegur
">
      <![CDATA[java.lang.AssertionError: 2 expectations failed.
Expected status code <400> but was <200>.

Expected header "Content-Type" was not "application/json", was "text/plain;charset=UTF-8". Headers are:
Server=Cowboy
Connection=keep-alive
Content-Type=text/plain;charset=UTF-8
Content-Length=50
Date=Fri, 18 Aug 2023 18:25:51 GMT
Via=1.1 vegur

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:101)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:288)
at stepDefinitions.TC03_PutReq_ById_Prog.user_receive_http_status_code_and_error_message(TC03_PutReq_ById_Prog.java:118)
at ✽.User receive HTTP Status code and error message(file:///C:/Users/nsuma/OneDrive/Desktop/Sample%20API/LMS_API_2023/src/test/java/features/TC03_PutProgram.feature:28)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="1.024">
    <failure type="java.lang.AssertionError" message="expected [400] but found [500]">
      <![CDATA[java.lang.AssertionError: expected [400] but found [500]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:907)
at org.testng.Assert.assertEquals(Assert.java:917)
at stepDefinitions.TC03_PutReq_ById_Prog.user_receive_http_status_code_and_error_message(TC03_PutReq_ById_Prog.java:122)
at ✽.User receive HTTP Status code and error message(file:///C:/Users/nsuma/OneDrive/Desktop/Sample%20API/LMS_API_2023/src/test/java/features/TC03_PutProgram.feature:34)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.356"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.356"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.724">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected header &amp;quot;Content-Type&amp;quot; was not &amp;quot;application/json&amp;quot;, was &amp;quot;text/plain;charset=UTF-8&amp;quot;. Headers are:
Server=Cowboy
Connection=keep-alive
Content-Type=text/plain;charset=UTF-8
Content-Length=50
Date=Fri, 18 Aug 2023 18:25:53 GMT
Via=1.1 vegur
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected header "Content-Type" was not "application/json", was "text/plain;charset=UTF-8". Headers are:
Server=Cowboy
Connection=keep-alive
Content-Type=text/plain;charset=UTF-8
Content-Length=50
Date=Fri, 18 Aug 2023 18:25:53 GMT
Via=1.1 vegur

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:101)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:288)
at stepDefinitions.TC05_DeleteReq_Prog.user_receives_status_code_with_success_message(TC05_DeleteReq_Prog.java:61)
at ✽.User receives Status Code with success message(file:///C:/Users/nsuma/OneDrive/Desktop/Sample%20API/LMS_API_2023/src/test/java/features/TC05_DeleteProgram_byID.feature:11)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.33">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected header &amp;quot;Content-Type&amp;quot; was not &amp;quot;application/json&amp;quot;, was &amp;quot;text/plain;charset=UTF-8&amp;quot;. Headers are:
Server=Cowboy
Connection=keep-alive
Content-Type=text/plain;charset=UTF-8
Content-Length=74
Date=Fri, 18 Aug 2023 18:25:53 GMT
Via=1.1 vegur
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected header "Content-Type" was not "application/json", was "text/plain;charset=UTF-8". Headers are:
Server=Cowboy
Connection=keep-alive
Content-Type=text/plain;charset=UTF-8
Content-Length=74
Date=Fri, 18 Aug 2023 18:25:53 GMT
Via=1.1 vegur

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:101)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:288)
at stepDefinitions.TC06_DeleteRq_ByName_Prog.user_receives_status_code_with_message(TC06_DeleteRq_ByName_Prog.java:33)
at ✽.User receives Status Code <200> with message(file:///C:/Users/nsuma/OneDrive/Desktop/Sample%20API/LMS_API_2023/src/test/java/features/TC06_DeleteProgram_byName.feature:11)
]]>
    </failure>
  </testcase> <!-- runScenario -->
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.711"/>
  <testcase classname="io.cucumber.testng.AbstractTestNGCucumberTests" name="runScenario" time="0.678">
    <failure type="java.lang.AssertionError" message="1 expectation failed.
Expected header &amp;quot;Content-Type&amp;quot; was not &amp;quot;application/json&amp;quot;, was &amp;quot;text/plain;charset=UTF-8&amp;quot;. Headers are:
Server=Cowboy
Connection=keep-alive
Content-Type=text/plain;charset=UTF-8
Content-Length=50
Date=Fri, 18 Aug 2023 18:25:56 GMT
Via=1.1 vegur
">
      <![CDATA[java.lang.AssertionError: 1 expectation failed.
Expected header "Content-Type" was not "application/json", was "text/plain;charset=UTF-8". Headers are:
Server=Cowboy
Connection=keep-alive
Content-Type=text/plain;charset=UTF-8
Content-Length=50
Date=Fri, 18 Aug 2023 18:25:56 GMT
Via=1.1 vegur

at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
at java.base/jdk.internal.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
at java.base/java.lang.reflect.Constructor.newInstanceWithCaller(Constructor.java:499)
at java.base/java.lang.reflect.Constructor.newInstance(Constructor.java:480)
at org.codehaus.groovy.reflection.CachedConstructor.invoke(CachedConstructor.java:73)
at org.codehaus.groovy.runtime.callsite.ConstructorSite$ConstructorSiteNoUnwrapNoCoerce.callConstructor(ConstructorSite.java:108)
at org.codehaus.groovy.runtime.callsite.AbstractCallSite.callConstructor(AbstractCallSite.java:277)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure.validate(ResponseSpecificationImpl.groovy:512)
at io.restassured.internal.ResponseSpecificationImpl$HamcrestAssertionClosure$validate$1.call(Unknown Source)
at io.restassured.internal.ResponseSpecificationImpl.validate(ResponseSpecificationImpl.groovy:101)
at io.restassured.internal.ValidatableResponseOptionsImpl.spec(ValidatableResponseOptionsImpl.java:288)
at stepDefinitions.TC10_Delete_req_batch.user_receives_status_code_with_success_message_in_response(TC10_Delete_req_batch.java:33)
at ✽.User receives Status Code <200>with success message in response(file:///C:/Users/nsuma/OneDrive/Desktop/Sample%20API/LMS_API_2023/src/test/java/features/TC10_Delete_Batch.feature:10)
]]>
    </failure>
  </testcase> <!-- runScenario -->
</testsuite> <!-- Default test -->
